index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alazar Esheetu</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Merriweather:ital,wght@0,400;1,300&family=Montserrat:wght@900&family=Roboto&family=Sacramento&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-contanier">
+        <div class="logo"><a href="#">Alazar Eshetu</a></div>
+        <div>
+          <img class="menu-icon" src="Images/Icon - Menu.png" alt="menu-icon" />
+        </div>
+      </nav>
+    </header>
+    <div class="headline">
+      <div><h1>Hey There. I’m Alazar</h1></div>
+      <div><p class="supporting-text">I’am a Software Developer</p></div>
+      <div>
+        <p class="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <ul class="link">
+        <li class="all raddit">
+          <img src="Images/raddit.png" alt="raddit-icon" />
+        </li>
+        <li class="all Linkdn">
+          <img src="Images/Linkdn.png" alt="linkedn-icon" />
+        </li>
+        <li class="all peace"><img src="Images/peace.png" alt="Icon" /></li>
+        <li class="all tewtter">
+          <img src="Images/twet.png" alt="tewtter-icon" />
+        </li>
+        <li class="all mail"><img src="Images/mail.png" alt="mail-" /></li>

```

Comment:
- [ ] [OPTIONAL] Kindly wrap the `img` tags in `a` tags and put your social links there. It would be great when someone clicks the icon, and boom he is already checking your LinkedIn profile. 😎️